tests/extract/test_extract_alerts.sql

```diff
@@ -1,143 +1,1037 @@
+SELECT *
```

Comment:
It looks like the formatting changed here... I'm curious if anything else did and why the format was changed?

Reply:
I think it's because me & Kyle used different PyCharm formatting settings.
He wasn't too bothered by formatting issues.

I made a commit so that both SQL files have the same format, as currently they're formatted differently.